depends/README.md

```diff
@@ -22,7 +22,7 @@ Common `host-platform-triplets` for cross compilation are:
 
 - `i686-w64-mingw32` for Win32
 - `x86_64-w64-mingw32` for Win64
-- `x86_64-apple-darwin11` for macOS
+- `x86_64-apple-darwin14` for macOS
```

Comment:
Does this change anything?

Reply:
Huh, how does this even compile on travis/gitian, given that support for 10.7 has been removed?

Reply:
I've never seen a case where this changes anything in practice. As a test, I just had a look at gcc's configure.ac though, and it at least checks for >=10 in a few places. So I guess we should keep these in sync to be on the safe side.

Reply:
Updated in travis.yml and gitian-osx.yml